db/migrate/20221216171738_value_overrides_to_non_canonical_profiles.rb

```diff
@@ -0,0 +1,12 @@
+class ValueOverridesToNonCanonicalProfiles < ActiveRecord::Migration[7.0]
+  class Profile < ApplicationRecord; end
+
+  def up
+    query = 'UPDATE profiles AS p
+             SET value_overrides = (SELECT value_overrides
+                                    FROM profiles
+                                    WHERE (p.parent_profile_id = id))
+             WHERE parent_profile_id IS NOT NULL'
```

Comment:
Isn't this a little bit too complicated? Also does it work as it should? What I found feasible (and shorter) is:
```
UPDATE "profiles" 
SET "value_overrides" = "parent_profiles"."value_overrides"
FROM profiles AS parent_profiles
WHERE "profiles"."parent_profile_id" IS NOT NULL AND "profiles"."parent_profile_id" = "parent_profiles"."id"
```